packages/data-sdk/package.json

```diff
@@ -26,7 +26,7 @@
   },
   "version": "0.0.147",
   "scripts": {
-    "dev": "vite --port 9146",
+    "dev": "vite --port 5173",
```

Comment:
Hmm, why the change in ports?

Reply:
I have so many custom modules/data-sdk/ui-sdk that I have to add to different orgs to test them out, and it is just a headache to go and manually >settings>modules>URL>url schema. It is just easy to have them in the same port so I can just add them to a view or to teleop and test.